src/util/fetchPokemonData.js

```diff
@@ -0,0 +1,12 @@
+const baseUrl = `https://pokeapi.co/api/v2/pokemon/`;
+
+export const fetchPokemonDataByName = async (name) => {
```

Comment:
Rename to something that includes both name and id search terms

Reply:
good catch!